h5io/_h5io.py

```diff
@@ -80,6 +82,10 @@ def write_hdf5(fname, data, overwrite=False, compression=4,
     title : str
         The top-level directory name to use. Typically it is useful to make
         this your package name, e.g. ``'mnepython'``.
+    slash : 'error' | 'replace'
+        Whether to replace forward-slashes ('\') in any key found nested within
+        keys in data. This does not apply to the top level name (title).
+        If 'error', '\' is not allowed in any lower-level keys.
```

Comment:
does this work? I think HD5 will warn about valid identifiers / key naming pattenrs, we should check before  ...